### PR TITLE
Da/dkg disabled flow

### PR DESCRIPTION
**Pull request by axelrod-blox**
Screenshot 2023-10-23 at 12 58 50

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export interface IOperator {
   selected?: boolean,
   dappNode?: boolean,
   ownerAddress: string,
+  dkg_address?: string,
```
Comment (meshin-dev) [inline src/app/common/stores/applications/SsvWeb/Operator.store.ts:38]: Can it be camel-case? 
The same question about verified_operator field

File: src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
+import {
+    useStyles,
+} from '~app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.styles';
+
+const DkgOperator = ({ operator }: { operator: IOperator }) => {
+    const classes = useStyles({ operatorLogo: operator.logo, dkgEnabled: operator.dkg_address });
```
Comment (meshin-dev) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx:9]: Can dkgEnabled flag be true by the fact of real validation if it's domain name /ip address with optional port?
If value is not validated - then it's not dkg enabled

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 3]
```diff
@@ -82,14 +108,41 @@ const OfflineKeyShareGeneration = () => {
...
+
+    const disabledCondition = () => {
+        if (selectedBox === 1 || selectedBox === 3 && operatorsAcceptDkg) {
+            return !textCopied;
+        } else if (selectedBox === 0) {
+            return true;
+        } else {
+            return false;
+        }
+    };
```
Comment (meshin-dev) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:134]: Can disabled condition be a state which changes depending on selected box and operatorsAcceptDkg?

**Review by meshin-dev — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx:9]: done

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts:38]: this fields came from api as underscore

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 4]
```diff
@@ -82,14 +109,41 @@ const OfflineKeyShareGeneration = () => {
...
 
+    const changeWithdrawalAddressHandler = (e: any) => {
+        const { value } = e.target;
+        setWithdrawalAddress(value);
+        validateAddressInput(value, setAddressValidationError);
+        setTextCopied(false);
+        setConfirmedWithdrawalAddress(false);
+    };
+
+    const showCopyButtonCondition = selectedBox === 1 || (selectedBox === 3 && withdrawalAddress && !addressValidationError.shouldDisplay && confirmedWithdrawalAddress);
```
Comment (meshin-dev) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:120]: Sorry I didn't see these numbers here, can you do here the same as in first PR (use enum or constants)

**Review by meshin-dev — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx (hunk 4 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx]: done

**Review by meshin-dev — APPROVED** [review]

**Pull request merged**